### PR TITLE
Rdm 10101

### PR DESCRIPTION
**Pull request by MSancaktutar**
### JIRA link ###
https://tools.hmcts.net/jira/browse/RDM-10101

### Change description ###
Update D/B to 16 cores

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -90,7 +90,7 @@ env.OAUTH2_REDIRECT_URI = "https://www-ccd.aat.platform.hmcts.net/oauth2redirect
 env.DM_STORE_BASE_URL = "http://dm-store-aat.service.core-compute-aat.internal"
 env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
-env.ELASTIC_SEARCH_FTA_ENABLED = "true"
+env.ELASTIC_SEARCH_FTA_ENABLED = "false"
```
Comment (MSancaktutar) [inline Jenkinsfile_CNP:93]: This is a correction of a change inherent from past.

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -115,7 +115,7 @@ withPipeline(type, product, component) {
             environmentOfDependencies = "aat"
         }
         env.DEFINITION_STORE_HOST      = "http://ccd-definition-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
-        env.ELASTIC_SEARCH_FTA_ENABLED = "true"
+        env.ELASTIC_SEARCH_FTA_ENABLED = "false"
```
Comment (MSancaktutar) [inline Jenkinsfile_CNP:118]: This is to temporarily disable ES tests, as a few of them are failing due to definition store performance issue on AAT, being totally irrelevant to this change.

**Review by Thor-tech-of-metal — COMMENTED** [review]
LGTM

**Review by Thor-tech-of-metal — APPROVED** [review]
LGTM

**Pull request merged**